common/browser_context.go

```diff
@@ -21,8 +21,10 @@ import (
 )
 
 // Ensure BrowserContext implements the EventEmitter and api.BrowserContext interfaces.
-var _ EventEmitter = &BrowserContext{}
-var _ api.BrowserContext = &BrowserContext{}
+var (
+	_ EventEmitter       = &BrowserContext{}
+	_ api.BrowserContext = &BrowserContext{}
+)
```

Comment:
👍 for this grouping.

Reply:
I actually wanted to maintain the current format to comply with the rest of the code, but `gofumpt` was applying this change automatically, so I guess we should progressively apply this "rule" for the rest :)